src/views/mtls/MtlsCreateView.tsx

```diff
@@ -0,0 +1,100 @@
+import { Button } from "../../components/Button";
+import { useGetCertificate } from "../../hook/api/useGetCertificate";
+import { Layout } from "../../components/Layout";
+import { useEffect, useState } from "react";
+import { getOperatingSystem } from "../../hook/helpers/getOperatingSystem";
+import { WindowsInstructions } from "./WindowsMtlsInstructions";
+import { MacInstructions } from "./MacMtlsInstructions";
+import { LinuxInstructions } from "./LinuxMtlsInstructions";
+import { AndroidInstructions } from "./AndroidMtlsInstructions";
+import { IosInstructions } from "./IosMtlsInstructions";
+import { CardsContainer } from "../../components/CardsContainer";
+import { Text } from "../../components/Text";
+import { InfoModal } from "../../components/InfoModal";
+import key from "../../assets/icons/key.svg";
+
+export function MtlsCreateView() {
+  const { mutate: downloadCert } = useGetCertificate();
+  const [callsign, setCallsign] = useState("");
+  const [userOS, setUserOS] = useState("");
```

Comment:
Is there reason for state?
Could we memoize it in useOperationSystem hook and use that?